File: 2. Valid Anagram (242)/Valid Anagram (242).py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        a=[0]*26
        if len(s)!=len(t):
            return False
        for x in range(0, len(s)):
            a[ord(s[x])-ord('a')]+=1
            a[ord(t[x])-ord('a')]-=1
        for x in range(0,26):
            if a[x]!=0:
                return False
        return True

File: 2. Valid Anagram (242)/Valid Anagram (242).cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        if(s.size()!=t.size()) return false;
        int a[26]={};
        for(int i=0; i<26; ++i) cout<<a[i]<<"\n";
        for(int i=0; i<s.size(); ++i){
            ++a[s[i]-'a'];
            --a[t[i]-'a'];
        }
        for(int i=0; i<26; ++i) if(a[i]!=0) return false;
        return true;
    }
};